src/wallet/init.cpp

```diff
@@ -220,6 +221,19 @@ bool WalletInit::ParameterInteraction()
         LogPrintf("%s: parameter interaction: can't use -hdseed and -mnemonic/-mnemonicpassphrase together, will prefer -seed\n", __func__);
     }
 
+    if (gArgs.IsArgSet("-privatesenddenoms")) {
+        int nDenomsDeprecated = gArgs.GetArg("-privatesenddenoms", DEFAULT_PRIVATESEND_DENOMS_HARDCAP);
+        InitWarning("Warning: -privatesenddenoms is deprecated, please use -privatesenddenomshardcap instead.\n");
```

Comment:
should probably mention privatesenddenomsbatched here to. Something like "please instead use -privatesenddenomshardcap or -privatesenddenomsgoal depending on your purpose"

Reply:
80dff12